docs/glossary.md

```diff
@@ -16,7 +16,7 @@ The thread that controls a collection of related web content. This could be thou
 
 ### Display list ###
 
-TODO
+A list of concrete rendering instructions. The display list is post-layout, so all items have absolute pixel positions, and z-index has already been applied, so items later in the display list will always be on top of items earlier in it.
```

Comment:
Items in the display list are indeed relative to their parent stacking context and do not include any transformations applied by parents.